resources/views/reports/weekly_report.blade.php

```diff
@@ -8,6 +8,7 @@
             <a href="/reports/weekly-report?year={{ $year + 1 }}">Next</a>
         </div>
         <div class="box mt-3">
+            <div class="box__section box__section--header">Weekly Balance</div>
```

Comment:
The heading on line 5 already states that it's weekly. If you want to specify what we're showing, I'd suggest just stating "Balance" or something 👍

Reply:
Will think of a better name. Thanks